smac/runhistory/runhistory.py

```diff
@@ -293,7 +293,7 @@ def get_cost(self, config: Configuration):
         cost: float
             Computed cost for configuration
         """
-        config_id = self.config_ids[config]
+        config_id = self.config_ids.get(config)
```

Comment:
What's the reason for this change?

Reply:
When attempting to get the cost of a config that had not been evaluated, this function raised a key error. So using `get()` returns None if a key is missing. Also, this makes it consistent across other `get_*` methods from RunHistory.